### DISCUSSION TITLE
spring-boot-2/10-1

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#18] [Announcements]
# spring-boot-2/10-1

为什么加了@Transactional注解，事务没有回滚？ 在昨天的《事务管理入门》一文发布之后，有读者联系说根据文章尝试，加了@Transactional注解之后，事务并没有回滚。 经过一顿沟通排查之后，找到了原因，在此记录一下，给后面如果碰到类似问题的童鞋一个参考。 问题原因

https://www.didispace.com/spring-boot-2/10-1.html

**Comment by joeljhou** [comment]
hello!
